fix 增加悬浮窗的英文文档 (#859)
taptap/tds-docs: i18n/en/docusaurus-plugin-content-docs/current/sdk/taptap-connect/features.mdx

---
title: TapTap Connect Features
sidebar_label: Features
sidebar_position: 1
---

## Products

> TapTap Connect (hereafter referred to as the Floating Window) is a tool that connects TapTap, games and players. Once a developer has enabled this feature, players who have successfully signed in to their TapTap accounts can browse game news, rate games, share game download links, and more in the Floating Window to help games improve retention and attract new users.

Reviewer: ```suggestion
TapTap Connect (hereafter referred to as the Floating Window) is a tool that connects TapTap, games and players. Once a developer has enabled this feature, players who have successfully logged in their TapTap accounts can browse game updates, rate games and share game download links in the Floating Window to help improve retention and bring in new users.

```

![](https://capacity-files.lcfile.com/NiPOqorbHRaoXCJvgoAmJzXXvKwJnMCU/461703147053_.pic_hd.jpg)

## Player Interaction Flow

> - After logging into the game with TapTap, the player will see a Floating Window portal, which is semi-hidden by default and supports drag-and-drop and hovering.

Reviewer: ```suggestion
- After logging into the game with TapTap, a Floating Window portal will appear, which is semi-hidden by default and supports drag and drop as well as hovering.
```

> - Clicking on the portal will open the Floating Window panel, which displays the player's TapTap account information (avatar, nickname and ID), and also allows the player to open inline news, share the game, and jump to the TapTap game details page to rate the game from within the Floating Window. (The rating feature is only supported by the Android system.)

Reviewer: ```suggestion
- Clicking on the portal will open the Floating Window panel, which displays the player's TapTap account information (avatar, nickname and ID), and also allows the player to open embedded updates, share games, and jump to the TapTap game page to rate the game from within the Floating Window. (The rating feature is only supported by the Android system.)
```

> - Players who click on the blank space outside the Floating Window panel will close the Floating Window and the lobby will automatically return to a semi-hidden state after 3 seconds.

Reviewer: ```suggestion
- Players who click on the blank space outside the Floating Window panel will close the Floating Window and the portal will automatically return to a semi-hidden state after 3 seconds.
```

## Functional Advantages

> TapTap community content, rating sharing and other features are integrated so that players do not have to leave the game to perform related operations.

Reviewer: ```suggestion
TapTap community content, review sharing and other capabilities are integrated so that players can perform related operations without leaving the game.
```

> - Firstly, it can help the game to increase retention and attract new users;

Reviewer: ```suggestion
- Firstly, it can help the game increase retention and bring in new users;
```

> - Secondly, it reduces the amount of access development for developers. After accessing the Floating Window, there is no need to separately access inline dynamics, open the comment area, share and other features.

Reviewer: ```suggestion
- Secondly, it reduces the amount of access development for developers. After accessing the Floating Window, there is no need to separately access embedded dynamics, open the comment area, share and other features.
```

- Clicking on the portal will open the Floating Window panel, which displays the player's TapTap account information (avatar, nickname and ID), and also allows the player to open embedded updates, share games, and jump to the TapTap game page to rate the game from within the Floating Window. (The rating feature is only supported by the Android system.)
- Players who click on the blank space outside the Floating Window panel will close the Floating Window and the portal will automatically return to a semi-hidden state after 3 seconds.

## Functional Advantages

TapTap community content, review sharing and other capabilities are integrated so that players can perform related operations without leaving the game.

- Firstly, it can help the game increase retention and bring in new users;

- Secondly, it reduces the amount of access development for developers. After accessing the Floating Window, there is no need to separately access embedded dynamics, open the comment area, share and other features.

## Instructions for use

### Pre-Access Operation

> - The game needs to access the TapTap Login feature and enable the embedded dynamic service.

Reviewer: ```suggestion
- The game needs to access the TapTap Login feature and enable the embedded moments service.
```

- Enable the Floating Window feature in the [Game Services - TapTap Login - Floating Window ] module in the Developer Centre (the Floating Window feature is disabled by default and must be enabled by operation before it is displayed to players).

## Theme Style Configuration Instructions

> ### 1、Supports customisation of the Floating Window entry

Reviewer: ```suggestion
### 1、Supports customisation of the Floating Window portal
```

> - Defaults to using the icon provided by the platform

Reviewer: ```suggestion
- Using the icons directly provided by the platform by default.
```

- TapTap avatar can be used directly as the entrance icon.

- Developers should follow the design specification and design their own entrance icon to match the game.

> ### Supports customisation of the background of the user information area of the Floating Window panel.

Reviewer: ```suggestion
### Supports customizing the background of the user information area of the Floating Window panel
```
